src/components/Calculator.js

```diff
@@ -0,0 +1,29 @@
+import './Calculator.css';
+
+function Calculator() {
+  return (
+    <div className="calc-container">
+      <div className="calc-display">0</div>
+      <button type="button" className="gray-btn">AC</button>
+      <button type="button" className="gray-btn">+/-</button>
+      <button type="button" className="gray-btn">%</button>
+      <button type="button" className="orange-btn">/</button>
```

Comment:
- [ ] The `Calculator` component should be class-based, as per the project's requirement. Kindly refactor your code to make `Calculator` a React class component.

Reply:
Addressed this!